pilot/pkg/proxy/envoy/v1/testdata/rds-router-gateway-weighted-server1.json.golden

```diff
@@ -46,4 +47,4 @@
     ]
    }
   ]
- }
+ }
```

Comment:
(not important/nit) it's strange that just that one doesn't have a newline suddenly ?
I suppose it must mean the newlines are ignored for purpose of pass/fail ?

Reply:
this is what gets generated.